core/domain/exp_jobs_one_off.py

```diff
@@ -398,7 +398,11 @@ def map(item):
         if item.deleted:
             return
 
-        exploration = exp_fetchers.get_exploration_from_model(item)
+        try:
+            exploration = exp_fetchers.get_exploration_from_model(item)
+        except Exception as e:
+            yield ('Error %s when loading exploration' % str(e), [item.id])
```

Comment:
is there any chance of having `Exp Id:` as a part of `e`? (although there is a high chance of the job to not fall in this exception.

Reply:
Exp id is already in the end of message [loading exploration (item.id)], so I did not make any changes here. I am not sure if I understood your concern correctly, are you suggesting to add an Exp id prefix here?

Reply:
I mean is there any chance of `str(e)` to contain `Exp Id:`? (Note: we are parsing key and looking up for `Exp Id:` in the reduce method.) 

Example: `SomeError: Inavlid content id: Exp Id:`.

Reply:
Yes, we parsing the error key but if it is not found (`if exp_id_index == -1:`), we do not make any changes. So, it is fine to not have exp_id here in str(e). It is already present in the complete string so we don't need to include it again. In other cases, errors are returned as a list of strings in which we finally append the exp_id also.

Reply:
I was thinking of the case where there can be `Exp Id:` in the `str(e)`, in that case, the parser will divide the `str(e)` at an unexpected place and that would make the result untidy. But there is very less chance for this to happen! (I'm going to merge this!)

Reply:
Oh Ok, Got your point. Yes, but you are right that there is a less chance for that to happen. Thanks for merging this one!